rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -467,254 +772,76 @@ namespace rocal{
         m.def("VideoDecoderResize",&rocalVideoFileResize,"Reads videos from the source given and decodes it according to the policy only for Videos as inputs. Resizes the decoded frames to the dest width and height.",
             py::return_value_policy::reference);
         m.def("SequenceReader",&rocalSequenceReader,"Creates JPEG image reader and decoder. Reads [Frames] sequences from a directory representing a collection of streams.",
-            py::return_value_policy::reference,
-            py::arg("context"),
-            py::arg("source_path"),
-            py::arg("color_format"),
-            py::arg("shard_count"),
-            py::arg("sequence_length"),
-            py::arg("is_output"),
-            py::arg("shuffle") = false,
-            py::arg("loop") = false,
-            py::arg("frame_step"),
-            py::arg("frame_stride"));
+            py::return_value_policy::reference);
         m.def("rocalResetLoaders",&rocalResetLoaders);
         // rocal_api_augmentation.h
         m.def("SSDRandomCrop",&rocalSSDRandomCrop,
-            py::return_value_policy::reference,
-            py::arg("context"),
-            py::arg("input"),
-            py::arg("is_output"),
-	        py::arg("p_threshold"),
-            py::arg("crop_area_factor") = NULL,
-            py::arg("crop_aspect_ratio") = NULL,
-            py::arg("crop_pos_x") = NULL,
-            py::arg("crop_pos_y") = NULL,
-            py::arg("num_of_attempts") = 20);
-        m.def("Resize",&rocalResize, py::return_value_policy::reference);
+            py::return_value_policy::reference);
+        m.def("Resize",&rocalResize, 
```

Comment:
If we are changing all these, all of them can be in single line right?
`m.def("SSDRandomCrop",&rocalSSDRandomCrop, py::return_value_policy::reference);`

Reply:
If I do this, we dont see the differences between the old file and new file. It shows as a one big change, hence

Reply:
Ok got it